src/js/action/authAction.js

```diff
@@ -0,0 +1,11 @@
+
+import Dispatcher from '../dispatcher/Dispatcher';
+
+const getAuthStatus = (status) => {
+  Dispatcher.dispatch({
+    Type: 'AUTH',
```

Comment:
I think you could list all your action types as constant Strings and just refer to them where they are needed, that way you reduce the chances of bugs caused by typos.